backend/core-api/src/modules/auth/routes/oauth/utils.ts

```
@@ -0,0 +1,354 @@
+    const current = await redis.incr(key);
+
+    if (current === 1) {
+      await redis.expire(key, windowSecs);
+    }
+
+    if (current > limit) {
+      throw new RateLimitError('Too many requests. Please try again later.');
+    }
+  } catch (e) {
+    throw e;
+  }
```

ESLint

[error] Unnecessary try/catch wrapper.

(no-useless-catch)